source/camera/basic/capture_2d.py

```diff
@@ -8,8 +8,10 @@ def _main():
     camera = app.connect_camera()
 
     settings_2d = zivid.Settings2D()
-    settings_2d.iris = 50
-    settings_2d.exposure_time = datetime.timedelta(microseconds=50000)
+    settings_2d.iris = 11.31
```

Comment:
Zivid-python is based on 1.8 SDK. Therefore we can't use f-numbers but appropriate iris values. This is true for all samples.

Reply:
Fixed

Reply:
Fixed